src/include/nodes/plannerconfig.h

```diff
@@ -52,6 +52,8 @@ typedef struct PlannerConfig
 
 	bool		gp_cte_sharing; /* Indicate whether sharing is to be disabled on any CTEs */
 
+	bool is_under_subplan; /* True for plan rooted at a subquery which is planned as a subplan */
```

Comment:
I suppose `pgindent` should be run on all source code files, including headers.

Reply:
I corrected the code style.